core/src/test/scala/org/apache/spark/scheduler/BlacklistTrackerSuite.scala

```diff
@@ -40,7 +41,7 @@ class BlacklistTrackerSuite extends SparkFunSuite with BeforeAndAfterEach with M
     scheduler = mockTaskSchedWithConf(conf)
 
     clock.setTime(0)
-    blacklist = new BlacklistTracker(conf, clock)
+    blacklist = new BlacklistTracker(null, conf, clock)
```

Comment:
If this tracker is ever used, won't you get NPEs because the listener bus is null?

This basically means you're forcing every test to call `configureBlacklistAndScheduler()` manually. So either remove this initialization, or do this properly, and just call `configureBlacklistAndScheduler()` for tests that need to override the default config.

Since I don't see any test actually passing any custom config parameters, might as well do the initialization properly here and get rid of `configureBlacklistAndScheduler()` altogether.